neurons/miner.py

```diff
@@ -194,10 +195,6 @@ async def submit_model(self) -> None:
             subtensor=self.subtensor, netuid=self.config.netuid, wallet=self.wallet
         )
 
-        if len(self.config.hf_repo_type.encode('utf-8')) > 7:
```

Comment:
if we do fix the repo type to be "model" which is 5 characters long I guess we can adjust other validations to give them back 2 characters (the chain metadata can have up to 128 hence those checks to enforce input to fit in)